### DISCUSSION TITLE
Is it possible to use DB::raw in Laravel 8 upserts updateable fields?

### DISCUSSION THREAD

**Discussion by intergalactisch** [laravel/framework#35377] [Q&A]
Hi, 
I want to be able to increase a counter when a Category already exists in an import, e.g. so I can count how many products are in it. Something in the form of:
```
Category::upsert(
    $shopCategoryArray,
    ['name', 'shop_id'],
    [ 'product_count' => DB::raw('product_count + 1')]
);
```
Is this possible? It was in https://github.com/staudenmeir/laravel-upsert, whatever that means :D

**Comment by paras-malhotra** [comment]
Yes, it's possible. Anything that's possible in that package is possible in Laravel.
